### PR TITLE
docs: fix typos and add links (Ansible-Connection)

### PR DESCRIPTION
**Pull request by iambaboucarr**
Fixes typos and add link to ansible lxd plugin docs

### CODE REVIEW

File: docs/Ansible-Connection.md [hunk 1]
```diff
@@ -1,22 +1,22 @@
...
+The type of Architecture you are adopting determines which connection
+you'll be using. For example, if you're setting up dhis2 on an single server using
+lxd containers, your connection will be 'lxd'. However, Secure Shell (SSH) is the
+default connection plugin for ansible.
 
 ## lxd connection 
 When the setup is all in a single server, we separate components with lxd
 containers. To automate running tasks inside these containers with ansible,
-there is already a connection plugin available. Its lxd plugin. 
+there is already a connection plugin available. It's the [lxd plugin](https://docs.ansible.com/ansible/latest/collections/community/general/lxd_container_module.html).
```
Comment (tkipkurgat) [inline docs/Ansible-Connection.md:15]: I think you meant this link. https://docs.ansible.com/ansible/latest/collections/community/general/lxd_connection.html,

**Review by tkipkurgat — CHANGES_REQUESTED** [review]
Thanks for your contribution, @babouceesay 
I've done a quick review, and there is one suggested change on community.general.lxd connection link.

**Comment by babouceesay** [comment]
Thanks Tito for flagging that. I have applied the changes.

**Comment by tkipkurgat** [comment]
I can confirm that the requested changes have been made,

**Pull request merged**